Dockerfile.server

```diff
@@ -3,3 +3,8 @@ FROM orbs:base-server
 ADD . /opt/orbs
 
 RUN ./build-server.sh && yarn cache clean
+
+HEALTHCHECK --interval=10s --timeout=10s --retries=12 \
+    CMD curl -q http://localhost:8081/admin/startupCheck
+
+#CMD ./entrypoint.sh
```

Comment:
There is no `entrypoint.sh` and this piece is commented out.

Reply:
Removed

Reply:
Now I feel you mean that you want to have entrypoint.sh, so in that case let's talk tomorrow about what it should contain.